src/graph.js

```diff
@@ -40,37 +40,112 @@ class Graph {
   // Optionally accepts an array of other GraphNodes for the new vertex to be connected to
   // Returns the newly-added vertex
   addVertex(value, edges = []) {
-
+    const newNode = new GraphNode({ value, edges });
+    edges.forEach((edge) => {
+      edge.pushToEdges(newNode);
+    });
+    this.vertices.push(newNode);
+    if (this.vertices.length === 2) {
+      this.vertices[0].pushToEdges(this.vertices[1]);
+      this.vertices[1].pushToEdges(this.vertices[0]);
+    }
+    return newNode;
   }
   // Checks all the vertices of the graph for the target value
   // Returns true or false
   contains(value) {
-
+    let doesContain = false;
+    this.vertices.forEach((vert) => {
+      if (vert.value === value) {
+        doesContain = true;
+      }
+    });
+    return doesContain;
   }
   // Checks the graph to see if a GraphNode with the specified value exists in the graph 
   // and removes the vertex if it is found
   removeVertex(value) {
-
+    for (let i = 0; i < this.vertices.length; i++) {
```

Comment:
You'll also need to check here if edges need to be removed between the vertex being removed and any vertices it has shared edges with.